app/src/org/commcare/android/view/SquareButtonWithNotification.java

```diff
@@ -13,53 +13,56 @@
 import org.commcare.dalvik.R;
 
 /**
+ * Square button with subtext and notification text
+ *
  * @author Daniel Luna (dcluna@dimagi.com)
  */
 public class SquareButtonWithNotification extends RelativeLayout {
```

Comment:
I didn't look super closely at this, and I'm not sure how much real value it would add besides just neatness, but is there a reason that SquareButtonWithNotification can't also extend CustomButtonWithText?